src/index.html

```diff
@@ -1,22 +1,409 @@
+          ></a>
+        </div>
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#recommended"
+                >
+                  Recommended
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#categories"
+                >
+                  Categories
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#info"
+                >
+                  About Us
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#contact-us"
+                >
+                  Contact us
+                </a>
+              </li>
+            </ul>
+          </nav>
+          <a
+            href="tel:+1 234 55555-55-55"
+            class="menu__phone-number"
+          >
+            +1 234 55555-55-55
+          </a>
+          <a
+            href="tel:+1 234 55555-55-55"
+            class="menu__call-to-order"
+          >
+            CALL TO ORDER
+          </a>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="main__content">
+        <section
+          class="recommended"
+          id="recommended"
+        >
+          <h2 class="section-title">Recommended</h2>
+          <div class="recommended__products">
+            <article class="product recommended__product">
+              <img
+                src="images/products/beosound-a5.png"
+                alt="beosound-a5"
+                class="product__photo"
+              />
+              <h3 class="product__title">BEOSOUND A5</h3>
+              <p class="product__description">
+                Huge sound anywhere - in the home or outdoors. Beosound A5 is a
+                Wi-Fi and Bluetooth speaker with long-lasting battery and
+                wireless charging built in.
+              </p>
+              <p class="product__price">$ 1299.00</p>
+            </article>
+            <article class="product recommended__product">
+              <img
+                src="images/products/beoplay-hx.png"
+                alt="beoplay-hx"
+                class="product__photo"
+              />
+              <h3 class="product__title">BEOPLAY HX</h3>
+              <p class="product__description">
+                Wireless noise-cancelling headphones. Listen in comfort whether
+                you're on a call, working from home or taking a moment for
+                yourself.
+              </p>
+              <p class="product__price">$ 499.00</p>
+            </article>
+            <article class="product recommended__product">
```

Comment:
i only see hovers. Ur links don't work and `cursor: pointer` is missing.
Why do you use a completely different solution for similar products on the lower block(categories)?